src/plugins/xtriggers/index.rst

```diff
@@ -0,0 +1,53 @@
+Xtrigger Plugins
+======================================
+
+Xtrigger plugins allow you to install and use xtriggers without them being
+in your ``PYTHONPATH``.
+
+
+Built In Plugins
+----------------
+
+Cylc Flow provides the following xtriggers.
+
+.. autosummary::
+   :toctree: built-in
+   :template: docstring_only.rst
+
+   cylc.flow.xtriggers.echo
+   cylc.flow.xtriggers.wall_clock
```

Comment:
Unfortunately the fly in the ointment here is that the `wall_clock` function has a different signature in the code compared to real usage. Not sure what the reason for this was, but it was done in https://github.com/cylc/cylc-flow/pull/4511. Maybe @hjoliver will be able to shed some light

Reply:
I don't really understand this, outside my area of knowledge. Do I need to change anything or await advice from @hjoliver ?

Reply:
The `wall_clock` xtrigger isn't really an xtrigger because it would be inefficient to call this function in a subprocess over and over. So instead it's actually run synchronously within the main code, the trigger time is computed from the `offset` and the cycle point in advance to avoid excess computation.

This sort of problem will go away with https://github.com/cylc/cylc-flow/issues/3497 as this will avoid polling a function allowing it to maintain local state.

Until then, maybe just subtract the `wall_clock` function from the autodocumented reference.

Reply:
Thanks, removed wall_clock. I also noticed I had PYTHONPATH instead of CYLC_PYTHONPATH, so I updated that too.